src/integrations/GA4/utils.js

```diff
@@ -209,18 +190,30 @@ function getDestinationItemProperties(products, item) {
  * @param {*} props
  */
 function getPageViewProperty(props) {
-  return getDestinationEventProperties(
-    props,
-    pageEventParametersConfigArray,
-    "properties"
-  );
+  return getDestinationEventProperties(props, pageEventParametersConfigArray, 'properties');
+}
+
+/**
+ * Validates weather to send userId property to GA4 or not
+ * @param {*} integrations
+ */
+function sendUserIdToGA4(integrations) {
+  if (Object.prototype.hasOwnProperty.call(integrations, 'GA4')) {
```

Comment:
Can't we use the constant name for a destination? The customer can send `google analytics 4` as well right?

Reply:
@ItsSudip customer can use `google analytics 4` as well, we are transforming names to server one so it will come as `GA4` only. verified by using different names as well